.github/workflows/tests.yml

```diff
@@ -4,20 +4,21 @@ jobs:
   ci:
     name: CI
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # This should contain the Ruby versions of all projects that use this
+        # gem, so we can be sure the shared config will be valid everywhere.
+        # Due to https://github.com/actions/runner/issues/849, we have to use
+        # quotes for '3.0' -- without quotes, CI sees '3' and runs the latest.
+        ruby: [2.7, '3.0', 3.1]
     env:
       CI: true
     steps:
     - uses: actions/checkout@master
     - uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.7
-    - uses: actions/cache@v1
-      with:
-        path: vendor/bundle
-        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-gems-
```

Comment:
Is this `actions/cache@v1` section replaced by the `bundler-cache: true`?

Reply:
I believe so. I basically copied these changes over from [memo_wise](https://github.com/panorama-ed/memo_wise/blob/main/.github/workflows/main.yml#L29), without the comments.

Reply:
Ah yeah looks like this is a typical change: https://www.jessesquires.com/blog/2021/08/23/caching-bundler-on-github-actions/